yash-datalake/azuredeploy.json

```diff
@@ -213,7 +219,7 @@
     {
       "type": "Microsoft.Resources/deployments",
       "name": "storageAccount",
-      "apiVersion": "2019-09-01",
+      "apiVersion": "2019-08-01",

```

Comment:
why are we rolling back to older apiVersions?

Reply:
Newer API versions are creating problem right now

Reply:
What's the problem?  If there's a bug in the template or the platform we should address it...

Reply:
Actually this version was suggested in best practices check in devops pipeline so i used it but later when I checked in docs this version wasn't
So I just reverted to the next Lower version


https://docs.microsoft.com/en-us/azure/templates/microsoft.resources/2018-05-01/deployments